kvmagent/kvmagent/plugins/port_mirror_plugin.py

```
@@ -169,6 +171,35 @@ def _release_mirror_session_local(self, src_device_name, dst_device_name, direct
+        vlan_ifname = "%s.%d" % (interface_name, vlan_id)
+        if iproute.is_device_ifname_exists(vlan_ifname):
+            iproute.delete_link_by_name(vlan_ifname)
+
+    def _get_physical_target_device(self, mirror):
+        """获取物理机目标设备名称"""
+        if mirror.dnic:
+            return mirror.dnic
+        elif mirror.interfaceName:
+            return self._create_vlan_interface(mirror.interfaceName, mirror.vlanId)
+        else:
+            raise Exception("Either dnic or interfaceName must be specified")
```

Ruff (0.13.3)

Create your own exception

(TRY002)